KsApi/Data+PKCE.swift

```diff
@@ -0,0 +1,57 @@
+import CryptoKit
```

Comment:
Copy-pasted from the existing/previously reviewed code in `PKCE.swift`, just thought that file was getting a little long.